lib/ex_sdp/utils.ex

```diff
@@ -40,4 +40,30 @@ defmodule ExSDP.Utils do
       _string_not_0_nor_1 -> {:error, :string_not_0_nor_1}
     end
   end
+
+  @spec parse_sprop_parameter_sets(binary) ::
+          {:error, :invalid_sprop_parameter_sets} | {:ok, %{sps: binary, pps: binary}}
+  def parse_sprop_parameter_sets(string) do
+    result =
+      String.split(string, ",", parts: 2)
+      |> then(fn list -> List.zip([[:sps, :pps], list]) end)
```

Comment:
1. You can use `Enum.zip/2` that accepts 2 enumerables
2. There's no need to "swap" the key/value order since you map in next step

```suggestion
      |> Enum.zip([:sps, :pps])
```